util/get-toolchain.py

```diff
@@ -16,15 +16,27 @@
 
 log.basicConfig(level=log.INFO, format="%(levelname)s: %(message)s")
 
-ASSET_PREFIX = "lowrisc-toolchain-gcc-rv32imc-"
+# the keys in this dictionary specify valid toolchain kinds
+ASSET_PREFIXES = {
+    # kind : prefix,
```

Comment:
Nit: I find "kind" strange, "variant" would be better. But I realize you've used this terminology already in many places, so it's too late for that.